backend/btrixcloud/models.py

```diff
@@ -358,6 +358,8 @@ class RawCrawlConfig(BaseModel):
 
     userAgent: Optional[str] = None
 
+    selectLinks: Union[str, List[str], None] = "a[href]->href"
```

Comment:
Yeah, perhaps we just make it a list to simplify, as mentioned in the other comment.
```suggestion
    selectLinks: List[str] = ["a[href]->href"]
```